src/component/error/error.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+class Error extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            errorMeg: this.props.errorMeg
+        };
+      }
+      
+      render() {
+        return <span>
+        <img src="../../assets/images/images/error.png" />
```

Comment:
The error image is not showing.  Remember to import the image file as a variable and use the variable for your `src` attribute, so Babel can determine the file paths correctly.